sys/include/vfs.h

```diff
@@ -81,6 +81,70 @@ extern "C" {
 /* #define restrict */
 #endif
 
+/**
```

Comment:
I'm just wondering if this whole block shouldn't be in a separate file. But I'm OK to keep it like that, it could be revisited later on if there are new FS added.

Reply:
Since they are required by `VFS_DIR_BUFFER_SIZE` now (and `VFS_DIR_BUFFER_SIZE` needs to be updated when a new FS is added) I thought it's easier to have them both in the same file.

Reply:
I was more thinking about a file included here that would contain only this block. But that's fine like that.